config.json

```diff
@@ -27,16 +27,16 @@
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-ads",
-        "version": "19.6.0",
-        "nugetVersion": "119.6.0",
+        "version": "19.7.0",
+        "nugetVersion": "19.7.0",
```

Comment:
This is my fault because the script was wrong, but you will need to manually fix the `nugetVersion`s on all of these to be `+100`.  Future PR's created by the bot should be correct.

Reply:
There was more than 1. Can you verify I did check and fix all of them, please?